std/datetime/systime.d

```diff
@@ -446,6 +446,8 @@ private:
     calendar-based operations, and getting individual units from it such as
     years or days is going to involve conversions and be less efficient.
 
+    An $(I hnsec) (hecto-nanosecond) is 100 nanoseconds. There are 10,000,000 hnsecs in a second.
+
```

Comment:
https://github.com/dlang/druntime/blob/5a77d7fc0b3d3e1857a91915081ea26aa568de3c/src/core/time.d#L8-L14

Reply:
I don't think there's any issue having a little redundancy here

Reply:
I think it'd be better to link to core.time.hnsecs on the first use.